### PR TITLE
Added italian translation

### PR DESCRIPTION
**Pull request by sysadminmatmoz**
I closed the previous pull to commit a cleaner version.

### CODE REVIEW

File: project_gtd/i18n/it.po [hunk 1]
```diff
@@ -407,7 +410,6 @@ msgstr "o"
 #~ msgid "This Month"
 #~ msgstr "Questo mese"
 
-#, python-format
```
Comment (pedrobaeza) [inline project_gtd/i18n/it.po:410]: All of these strings are commented, so they're useless. Maybe a good idea would be to regenerate italian file, syncing previously all the terms.

**Comment by sysadminmatmoz** [comment]
Ok. I recreated the template (the existing one was kept since the version 6) and synced all the languages according to the new one. Now it should be clean.

**Comment by pedrobaeza** [comment]
Wow, what a huge work

:+1:

**Comment by sysadminmatmoz** [comment]
The question is if it has sense to keep such a number of translations since at least half of them is actually empty (translated less than 20%)

**Comment by pedrobaeza** [comment]
Well, less is nothing. We are moving to Transifex, so this work will soon be easy.

**Comment by sysadminmatmoz** [comment]
I know, I already translated in slovene nearly all OCA (v8) modules that are already there. :smile: and did the same with the Odoo ones (it's a lot to do if you are the "community" in your country).
But something should be done to ease the translation process by merging OCA and Odoo modules on transifex, otherwise the translated terms are not shared and you get cases, where the same term has a different translation in an Odoo module than it has in an OCA module since Transifex translation memory is not shared through different repositories as it was on launchpad - that's why I'm working on a custom poedit based platform in local - to have all the terms shared independently of the repo.

**Comment by pedrobaeza** [comment]
That's difficult to achieve due to the translation architecture itself. You can see an example when you need to translate the "Last Updated by" field and so on on each module that declares a new model.

**Comment by dreispt** [comment]
@pedrobaeza ...unless :bulb: an OCA script could translate empty strings copying from what was done on Odoo.

**Comment by pedrobaeza** [comment]
Yeah, maybe we can check and make any injection, but for now that's not feasible.

**Comment by dreispt** [comment]
:+1: 

By the way, this project [is already](https://www.transifex.com/oca/OCA-project-service-8-0/) on Transifex, but I'm not sure if the OCA Transifex sync is working.

**Comment by pedrobaeza** [comment]
Github > Transifex is already working. Transifex > Github is on the way here: https://github.com/OCA/maintainer-tools/pull/69

**Comment by sysadminmatmoz** [comment]
Should I close this PR?

**Comment by pedrobaeza** [comment]
No, it's OK for now to do it this way.

**Pull request merged**